go-apps/meep-gis-engine/api/swagger.yaml

```diff
@@ -320,6 +434,62 @@ definitions:
           - "CLOUD"
     - $ref: "#/definitions/GeoData"
     description: "List of geospatial data"
+  WithinRangeParameters:
+    type: "object"
+    required:
+    - "radius"
+    properties:
+      assetName:
+        type: "string"
+        description: "Asset name of a second element for query purpose. If present,\
+          \ latitude and longitude are ignored."
+      latitude:
+        type: "number"
+        format: "float"
+        description: "Latitude of a second element for query purpose. Taken into account\
+          \ only if AssetName is not present."
+      longitude:
+        type: "number"
+        format: "float"
+        description: "Longitude of a second element for query purpose. Taken into\
+          \ account only if AssetName is not present."
+      radius:
+        type: "number"
+        format: "float"
+        description: "Radius (in meters) around the location."
+      accuracy:
+        type: "number"
+        format: "float"
+        description: "Accuracy of the location (in meters)."
+    description: "Parameters for within range query purpose."
+    example:
+      latitude: 0.8008282
+      assetName: "assetName"
+      accuracy: 5.962134
+      radius: 1.4658129
+      longitude: 6.0274563
+  WithinRangeResponse:
```

Comment:
Again, recommend directly returing a _WithinRange_ object here.

Reply:
removed the Response part of the object name